android/app/src/main/res/layout/fragment_add_experience.xml

```diff
@@ -0,0 +1,251 @@
+            app:tint="@color/md_theme_primary" />
+
+        <TextView
+            android:id="@+id/tvPostExperience"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/post_experience"
+            android:textAppearance="@style/TextAppearance.Material3.TitleMedium"
+            android:textColor="@color/md_theme_onSurfaceVariant"
+            app:layout_constraintBottom_toBottomOf="@+id/fabBack"
+            app:layout_constraintEnd_toStartOf="@+id/fabPost"
+            app:layout_constraintStart_toEndOf="@+id/fabBack"
+            app:layout_constraintTop_toTopOf="@+id/fabBack" />
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/fabPost"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="0dp"
+            android:layout_marginEnd="15dp"
+            android:alpha="0.2"
+            android:backgroundTint="@color/md_theme_primary"
+            android:enabled="false"
+            android:src="@drawable/ic_done"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/fabBack"
+            app:shapeAppearanceOverlay="@style/RoundedCard"
+            app:tint="@color/md_theme_onPrimary" />
+
+        <com.google.android.material.card.MaterialCardView
+            style="?attr/materialCardViewElevatedStyle"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="20dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/fabBack">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <com.google.android.material.search.SearchBar
+                    android:id="@+id/sbMovie"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:hint="@string/search_movie"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <LinearLayout
+                    android:id="@+id/llNoSelectionPlaceholder"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:alpha="0.16"
+                    android:gravity="center"
+                    android:orientation="vertical"
+                    android:visibility="invisible">
```

Comment:
It is always invisible. In the code section I only see the state being change to gone but its never visible

Reply:
I'll check, maybe this is a mistake and should be visible